### PR TITLE
Fix HF section

### PR DESCRIPTION
**Pull request by messiisgreat**
## Checkout Points

- [ ] Check if renaming to PoC done
image

- [ ] Check if changing icons(5) done
- [ ] Check if adding HF to menu(in dropdown and footer) done
image

## Suggestion

- Suggest better approach

### CODE REVIEW

**Review by Maxnflaxl — CHANGES_REQUESTED** [review]
please check grafik

File: locales/en.json [hunk 1]
```diff
@@ -474,6 +474,7 @@
       "beamForum": "Beam Forum",
       "beamBlog": "Beam Blog",
       "roadmap": "Roadmap",
+      "hardfork": "Hardfork",
```
Comment (Maxnflaxl) [inline locales/en.json:477]:
```suggestion
      "hardfork": "Hardforks",
```

File: locales/en.json [hunk 2]
```diff
@@ -205,16 +205,15 @@
         }
       },
       "8": {
-        "title": "Groovy Gluon DeFi",
+        "title": "PoC",
```
Comment (Maxnflaxl) [inline locales/en.json:208]:
```suggestion
        "title": "Groovy Gluon DeFi",
```

**Review by Maxnflaxl — CHANGES_REQUESTED** [review]
add missing translations

File: utils/emitter.ts [hunk 3]
```diff
@@ -4,9 +4,11 @@ export enum UserInteractionEvents {
   CLOSE_MENUS = "closeMenus",
   SCROLL_TO_WHERE_TO_BUY = "scrollToWhereToBuy",
   SCROLL_TO_ROADMAP = "scrollToRoadmap",
+  SCROLL_TO_HARDFORK = "scrollToRoadmap",
```
Comment (Maxnflaxl) [inline utils/emitter.ts:7]: I think you have a double "scrollToRoadmap" here
```suggestion
  SCROLL_TO_HARDFORK = "scrollToHardfork",
```

**Review by Maxnflaxl — APPROVED** [review]

**Comment by Maxnflaxl** [comment]
can you send a screenshot of the 2023 and 2024 roadmap to confirm changes? Ty

**Comment by messiisgreat** [comment]
check this plz
image

**Review by Maxnflaxl — CHANGES_REQUESTED** [review]
add "Beam Messenger PoC" to 2023 and "Beam Messenger DApp" to 2024

**Comment by Maxnflaxl** [comment]
screenshot please ^^

**Comment by messiisgreat** [comment]
image

**Review by Maxnflaxl — APPROVED** [review]

**Pull request merged**